src/js/utilities/postHandling.mjs

```diff
@@ -0,0 +1,231 @@
+const API_BASE_URL = `https://api.noroff.dev/api/v1`;
```

Comment:
Like mentioned in the other file, this base url should be imported from a constants file so that it is set in one place, not many places.